config/initializers/rakismet.rb

```diff
@@ -0,0 +1,2 @@
+Rails.application.config.rakismet.key = ENV['RAKISMET_KEY']
+Rails.application.config.rakismet.url = ENV['RAKISMET_URL']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.